doc/wiki/notes/ch4_namespace.md

```diff
@@ -1,57 +1,50 @@
-This note explains how the namespaces should be used in CH4.
+# CH4 Namespace Convention
 
-= Terms in this note:
-  1. Component: a set of data structures and/or functions that related
-     to a particular functionality.
+This page discusses how namespaces should be used in CH4.
 
-= MPID_ namespace
-  This is the abstract device interface (ADI). Usually, these names are
-  linked to their corresponding implementations in the MPIDI_ namespace.
+Below is the format that namespaces will follow on this page:
+    1. `Component` - A set of data structures and/or functions that related
+       to a particular functionality.
```

Comment:
The original version is better. It explains a "term", not "format"

Reply:
Reverted.